opentracing_instrumentation/client_hooks/base.py

```diff
@@ -0,0 +1,25 @@
+class Patcher(object):
```

Comment:
can we call this file `_patcher.py` instead?

Reply:
I've renamed `base.py` to `_patcher.py`.